applications/metaquity_nft_provenance_tracker.md

```diff
@@ -0,0 +1,144 @@
+- Arko Ganguli
+
+### Contact
+
+- **Contact Name:** Abhishek Chatterjee
+- **Contact Email:** abhishek@metaquity.xyz
+- **Website:** [Metaquity](www.metaquity.xyz/)
+
+### Legal Structure
+
+- **Registered Address:** 128 CITY ROAD, UNITED KINGDOM, LONDON, EC1V 2NX
+- **Registered Legal Entity:** Metaquity Limited.
+
+### Team's experience
+
+- **Abhishek Chatterjee**: Co-founder of Metaquity, 15+ years of product and technical experience. 4+ years of blockchain experience. 
+- **Abhishek Sira Chandrasekhar**: Software developer with 4+ years of experience in development 2+ years in Blockchain working on DID, SSI, VC and ZKP. Prior to Metaquity used to work at Litentry as a mobile developer.
+
+- Metaquity Network is already part of the Substrate Builders Program.
+
+
+### Team Code Repos
+
+- https://github.com/metaquiry-network/nft-provenance-tracker
+
+Team members GitHub:
+
+- https://github.com/abhichat85
+- https://github.com/ekaanth
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/abhishekchatterjee85/
+- https://www.linkedin.com/in/abhishek-sira-chandrashekar-821244b7/
+
+## Development Roadmap :nut_and_bolt:
+
+This project will be a proof of concept (PoC) to demonstrate a particular use case and how NFT-SDK can be leveraged by other projects and parachains in the Polkadot ecosystem. Since this will be an SDK, anyone will be able to integrate this within their application. This acts as an enabler for this particular use case for provenance tracking of NFTs and fractional fungible tokens associated with those NFTs. This can extend to not only digital assets but also real-world assets (which is what is the primary focus of Metaquity Network).
+
+### Overview
+
+- **Total Estimated Duration:** 2,5 months
+- **Full-Time Equivalent (FTE):**  2 FTE (full-time employees)
+- **Total Costs:** 20,000 USD
+
+### Milestone
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation |  We will provide both readme.md and official architectural documentation of the NFT scanner module |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | Dockerfiles and docker-compose.yml for running a development environment that locally spins up, which will be able to interact with the database to fetch the indexed information locally. |
+| **0e.** | Article | Add an article covering how to integrate NFT-SDK in a project. |
+| 1. | NFT Schema Development | Development of data models for the indexing of the NFT-related extrinsic so that users can get compressive information of the NFT lifecycle  |
+| 2. | Collection schema development | Development of NFT-collection schema models |
+| 3. | NFT fractionalization schema development | Development of fractionalization schema models |
```

Comment:
What's the difference to the previous deliverable _2. | Collection schema development_ here?